src/tab/Tab.tsx

```diff
@@ -70,9 +72,9 @@ function Tab({
       <div className={"tab__body"} data-testid={`${testid}.body`}>
         {
           children[
-            activeTabIndexFromProps === undefined
-              ? activeTabIndex
-              : activeTabIndexFromProps
+          activeTabIndexFromProps === undefined
+            ? activeTabIndex
+            : activeTabIndexFromProps
```

Comment:
The indentation doesn't look correct,
content of `[]` should be deeper, why was it changed in the first place? 🤔 I believe we already have eslint rules for the toolkit, it should take care of this.

Same for the diff in `ControlledTabProps`